### PR TITLE
NFDIV-1586

### PR DESCRIPTION
**Pull request by velimd**
### JIRA link ###
https://tools.hmcts.net/jira/browse/NFDIV-1586

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/functional/features/joint-hub-page.feature [hunk 1]
```diff
@@ -57,3 +57,12 @@ Feature: Joint hub page
     And I login with applicant "2"
     And the page should include "The hearing will take place at Birmingham Civil and Family Justice Centre on 29 September 2013 at 3:30PM."
     And the page should include "You can view and download your 'certificate of entitlement for a conditional order'."
+
+    Given I set the case state to "AwaitingClarification"
+    When I click "Sign out"
+    And I login with applicant "1"
+    And the page should include "Either you or your husband can provide the information requested by the court"
```
Comment (benouaer) [inline src/test/functional/features/joint-hub-page.feature:64]: should this be then

File: src/test/functional/features/joint-hub-page.feature [hunk 2]
```diff
@@ -57,3 +57,12 @@ Feature: Joint hub page
...
+    When I click "Sign out"
+    And I login with applicant "2"
+    And the page should include "Either you or your wife can provide the information requested by the court"
```
Comment (benouaer) [inline src/test/functional/features/joint-hub-page.feature:67]: should this be then, as you're asserting the result of your action

**Review by benouaer — APPROVED** [review]

File: src/test/functional/features/joint-hub-page.feature (hunk 1 above)
Comment (velimd) [reply src/test/functional/features/joint-hub-page.feature:64]: ngl i barely check the structure, as long as the rest does what it does as cucumber gets the function anyways.

Comment (velimd) [reply src/test/functional/features/joint-hub-page.feature:64]: ill still update it tho

**Pull request merged**